### PR TITLE
feat: add metabase component

### PR DESCRIPTION
**Pull request by revolunet**

### CODE REVIEW

File: src/components/metabase/index.test.ts [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+
+beforeEach(() => {
+  jest.resetModules();
+});
+
+test("should create metabase dev config", () => {
+  const env = new Environment("/tmp");
+  process.env.CI_REGISTRY_IMAGE = "/path/to/docker/image";
+  process.env.CI_ENVIRONMENT_URL = "https://path/to/env";
+  process.env.CI_PROJECT_NAME = "some-project";
```
Comment (douglasduteil) [inline src/components/metabase/index.test.ts:17]: Used in the code ?
I do not find this on the snapshot

File: src/components/metabase/index.ts [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+    env: Environment;
+    config?: Partial<AppConfig>;
+    deployment?: Partial<DeploymentParams>;
+  }
+): CreateResult =>
+  createApp("metabase", {
+    config: merge(
+      {
+        containerPort: 3000,
+        image: "metabase/metabase:v0.37.4",
```
Comment (douglasduteil) [inline src/components/metabase/index.ts:26]: Might want to use a renovate friendly version code

**Review by douglasduteil — APPROVED** [review]
👍

File: src/components/metabase/index.ts (hunk 2 above)
Comment (revolunet) [reply src/components/metabase/index.ts]: good ?

**Pull request merged**